docs/user-guide/options.md

```diff
@@ -212,6 +212,12 @@ CLI flags: `--report-needless-disables, --rd`
 
 Report `stylelint-disable` comments that don't actually match any lints that need to be disabled.
 
+## `reportUnscopedDisables`
+
+CLI flags: `--report-unscoped-disables, --rud`
+
+Report [configuration comments](../user-guide/ignore-code.md#parts-of-a-file) that aren't scoped to one or more rules.
```

Comment:
[note] Using "configuration comments" link instead of "`stylelint-disable` comments" is better. 👍🏼 

If you are interested, can you replace other "`stylelint-disable` comments" with such "configuration comments" links in this document on another pull request?

Reply:
> If you are interested, can you replace other "stylelint-disable comments" with such "configuration comments" links in this document on another pull request?

Ill do it in this PR.